sled-agent/src/sim/http_entrypoints_pantry.rs

```diff
@@ -23,13 +23,19 @@ pub fn api() -> CruciblePantryApiDescription {
     fn register_endpoints(
         api: &mut CruciblePantryApiDescription,
     ) -> Result<(), ApiDescriptionRegisterError> {
+        api.register(pantry_status)?;
+        api.register(volume_status)?;
         api.register(attach)?;
+        api.register(attach_activate_background)?;
+        // api.register(replace)?;
```

Comment:
I know it's in the commit message, but it would probably be useful to also have a comment here about why this is commented out.

Reply:
I took them out in 08786cca0, I now think this just clutters up the function